tests/integration/workflows/dotnet_clipackage/testdata/CustomRuntime6/aws-lambda-tools-defaults.json

```diff
@@ -12,5 +12,5 @@
   "function-memory-size": 256,
   "function-timeout": 30,
   "function-handler": "bootstrap",
-  "msbuild-parameters": "--self-contained true"
+  "msbuild-parameters": "--self-contained false"
```

Comment:
Does this mean customers might need to change this configuration on their end?

Reply:
Yes. Because we were always overriding msbuild-parameters at the command line, these properties were never actually used. So basically, users couldn't use msbuild-parameters in this file before, but now they can. I think about this more as fixing a bug or limitation.

But it could cause backward compatibility issues if someone had random msbuild parameters in here that they didn't actually want because now those parameters will actually get applied. 

So if we'd rather keep the somewhat limiting functionality but ensure backwards compatibility, we can probably keep passing both msbuild parameters and the new function-architecture arguments to the command line. 

Would you like us to revert it back to always passing msbuild-parameters down?